src/pyramid_debugtoolbar/panels/templates/session.dbtmako

```diff
@@ -0,0 +1,90 @@
+<h5>Session Configuration</h5>
+% if configuration:
+	<p>Using `ISessionFactory`: <code>${configuration}</code></p>
+% else:
+	<p>No `ISessionFactory` Configured</p>
+% endif
+
+
+## Panel Active
+% if is_active:
+	<p>This panel <span class="label label-success">was active</span> for this
+	request, and will always display the "Session" data -- even if `.session`
```

Comment:
I prefer short sentences for easier comprehension.

```suggestion
	request. It will always display the session data, even if <code>.session</code>
```

Reply:
Also apply to `else` clause below.